### ISSUE TITLE
stb_image.h: Comparison of unsigned expression >= 0 is always true

### ISSUE THREAD

**Issue by kaadmy** [nothings/stb#751]
Getting some warnings while compiling with `-std=c99 -Wextra -Wall -pedantic`

```
.../stb_image.h: In function ‘stbi__shiftsigned’:
.../stb_image.h:5113:18: warning: comparison of unsigned expression >= 0 is always true [-Wtype-limits]
    STBI_ASSERT(v >= 0 && v < 256);
                  ^~
.../stb_image.h:5113:4: note: in expansion of macro ‘STBI_ASSERT’
    STBI_ASSERT(v >= 0 && v < 256);
    ^~~~~~~~~~~
.../stb_image.h:5113:18: warning: comparison of unsigned expression >= 0 is always true [-Wtype-limits]
    STBI_ASSERT(v >= 0 && v < 256);
                  ^~
.../stb_image.h:5113:4: note: in expansion of macro ‘STBI_ASSERT’
    STBI_ASSERT(v >= 0 && v < 256);
    ^~~~~~~~~~~
```

**Comment by fluffrabbit** [comment]
Similar issue here:

    include/stb_image.h:5113:18: warning: comparison of unsigned expression >= 0 is always true [-Wtype-limits]
        STBI_ASSERT(v >= 0 && v < 256);
                      ^

**Comment by rygorous** [comment]
This was fixed in commit 48ffc6bc5576a716c5c105f7fb4f8b8d7ea45357 (shipped in version 2.22).
